README.md

```diff
@@ -16,7 +16,9 @@
 
 Sequelize is a promise-based Node.js ORM for Postgres, MySQL, MariaDB, SQLite and Microsoft SQL Server. It features solid transaction support, relations, eager and lazy loading, read replication and more.
 
-Sequelize follows [SEMVER](http://semver.org). Supports Node v6 and above to use ES6 features.
+Sequelize follows [SEMVER](http://semver.org). Supports Node v6 and above to use ES6 features. [Official TypeScript typings are also included](http://docs.sequelizejs.com/manual/typescript).
```

Comment:
This conflicts a little with my PR: https://github.com/sequelize/sequelize/pull/10677

Reply:
@SimonSchick oh, nice PR :grimacing: I've retracted this part of mine then.

But, I think you should update your PR with some mention to TypeScript more closer to the beginning of the README. Supporting TypeScript is a huge plus that IMO should be mentioned ASAP. Newcomers that take a glance at the README should see that TS is supported :grimacing: What do you think?